EVE.pm

```diff
@@ -96,7 +96,7 @@ sub new {
   $self->expand_left(0);
   $self->expand_right(0);
 
-  my $assembly = $self->{config}->{assembly};
+  my $assembly = $self->{config}->{assembly} || $config->{human_assembly};
```

Comment:
`$config` will not be defined here. Can you check?

Reply:
Thanks @msmanoj. Done!